wordpress-mobile/WordPress-iOS
# Login: Add new help alert for site address

Refs #7272

Adds the new site address help alert.

![simulator screen shot jul 9 2017 11 02 02 pm](https://user-images.githubusercontent.com/517257/28003879-084b6f5a-64fe-11e7-8590-40c81ec2f201.png)

To test:
- during login, tap "Log into your site by entering your site address instead."
- then tap "Need help finding your site address?"
- the new alert should appear.

### Note:
The design is more heavily based on the Aztec alert than the original Login design. For that reason I'm tagging @mattmiklic & @folletto for their feedback.

Needs review: @frosty 
cc: @aerych
